website/siteConfig.js

```diff
@@ -42,7 +42,7 @@ const siteConfig = {
     { doc: 'cumulus-docs-readme', label: 'Developer Docs' },
     { doc: 'data-cookbooks/about-cookbooks', label: 'Data-Cookbooks' },
     { doc: 'operator-docs/about-operator-docs', label: 'Operator Docs' },
-    { doc: 'team', label: 'Team' },
+    //{ doc: 'team', label: 'Team' }, Looks like team.md was removed in commit #1738
```

Comment:
Master is currently broken.   Two PRs up to address this, depending on what the team wants -- https://github.com/nasa/cumulus/pull/1747  or https://github.com/nasa/cumulus/pull/1748.

Reply:
Oh okay. There was a merge that went weird and I kept getting an error regarding team.md.